docs/Users/Tutorials/Testing_Bears.rst

```diff
@@ -0,0 +1,134 @@
+How to use LocalBearTestHelper to test your bears
+=================================================
+
+**coala** has an awesome testing framework so you can write tests for bears
```

Comment:
apparently, from what @sils1297 said, we don't bold coala, due to typographic disruption, so we decided to italic it.